cypress/integration/homeTest.js

```diff
@@ -1,11 +1,15 @@
 import HomePage from '../../pages/HomePage'
 import * as data from '../../test_data/home_data'
```

Comment:
Regarding `data` import.
You need to name your import more detailed:
`import * as homeData from '../../test_data/home_data'`
E.g. case when you'll need to add more tests in current file, and you'll import one more 'data' file.